File: Arrays/Last1.java
// { Driver Code Starts
//Initial Template for Java

//Initial Template for Java

import java.util.*;
import java.lang.*;
import java.io.*;

class GFG {
	public static void main(String[] args) throws IOException
	{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int t = Integer.parseInt(br.readLine().trim());
        while(t-->0)
        {
            String s = br.readLine();
            
            Solution obj = new Solution();
            System.out.println(obj.lastIndex( s));
            
        }
	}
}
// } Driver Code Ends


//User function Template for Java


class Solution {
    public int lastIndex( String s) {
        
        int size=s.length();
        int index=-1;
        for( int i=size-1;i>=0;i--)
        {
            if(s.charAt(i)=='1')
            {index=i; break;}
        }
        
        return index;
        
        
        
    }
}

File: Arrays/Last1.py
#User function Template for python3

class Solution:
    def lastIndex(self, s):
        # code here
        index=-1
        for i in range(len(s)-1,-1,-1):
            if(s[i]=='1'):
                index=i
                break
                
        return index
        
        


#{ 
#  Driver Code Starts
#Initial Template for Python 3


def main():

    T = int(input())

    while(T > 0):
    	s = input()
    	ob = Solution()
    	print(ob.lastIndex(s))
    	
    	T -= 1


if __name__ == "__main__":
    main()

# } Driver Code Ends
